CHANGELOG.md

```diff
@@ -8,6 +8,20 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 For compatibility information see `govukFrontendVersion` and `hmrcFrontendVersion` in
 [LibDependencies](project/LibDependencies.scala)
 
+## [5.4.0] - 2023-01-04
```

Comment:
@JoPintoPaul version number needs bumping

Reply:
@oscarduignan Thank you! Bumped and pushed.